### PR TITLE
describe upgrade 12 > 15; plus layout/markup fixes

### PR DESCRIPTION
**Pull request by keichwa**
I completely replaced 11 to 12 upgrade; it's pretty similar to the 12 to 15 upgrade (I could add a note, if wanted).

I also have to add a tip about "modules".

I still must check the autoyast example file.

### CODE REVIEW

File: adoc/bp_client_migration.adoc [hunk 1]
```diff
@@ -26,7 +27,7 @@ ifdef::env-github,backend-html5,backend-docbook5[]
 :susemgrproxy: SUSE Manager Proxy
 :productnumber: 3.2
 :saltversion: 2018.3.0
-:webui: WebUI
+:webui: Web UI
```
Comment (Loquacity) [inline adoc/bp_client_migration.adoc]: Oooh, nice one :)

File: adoc/bp_client_migration.adoc [hunk 2]
```diff
@@ -47,192 +50,176 @@ endif::[]
 :sourcedir: .
 :imagesdir: images
 
-{productname} supports you with upgrading SLE clients to the latest version or SP (service pack).
-Upgrading from SLE 12, SLE 12 SP1, or SP2 to SLE 12 SP3 or later (when available) is straight forward.
-Upgrading from SLE 11 SPx to SLE 12 SP3 or later (when available) requires some additional steps, but can be automated.
+{productname} supports you with upgrading SLE clients.
```
Comment (Loquacity) [inline adoc/bp_client_migration.adoc]: Can probably get rid of this line, it's not adding value.

File: adoc/bp_client_migration.adoc [hunk 3]
```diff
@@ -47,192 +50,176 @@ endif::[]
...
-
-[[bp.client.migration.sle11]]
-== Migrating {sle}  11 SPx to version 12 SP3
-
-{sle}
- 11 SPx clients may be auto-upgraded to version 12 SP3 by means of {yast}
-auto-installation.
-The same applies for other supported products based on {sle}
- 11.
+{sle} 12 SPx clients may be auto-upgraded to version 15 with {yast} auto-installation.
```
Comment (Loquacity) [inline adoc/bp_client_migration.adoc]: Is {yast} defined?

File: adoc/bp_client_migration.adoc [hunk 4]
```diff
@@ -47,192 +50,176 @@ endif::[]
...
-As you can see, it is especially important that you migrate the client to the latest available SP first (December 2017: Version 11 SP4).
+For generally supported {sle} upgrade paths, see  https://www.suse.com/documentation/sles-15/book_sle_upgrade/data/sec_upgrade-paths_supported.html ({slea} {upgrade}, Chapter "`Supported Upgrade Paths to SLE 15`").
++
+As you can see, it is important that you migrate the client to the latest available SP first.  In the case of SLE 12, this is currently SP3 (August 2018).
 ====
 
 
-This procedure will work for "`traditionally`"
- managed systems (system type ``management``).  Using Salt it will not be available (system type ``salt``).
+This procedure will work for "`traditionally`"  managed systems (system type ``management``).  Using Salt (system type ``salt``), for the time being, it will not be available.
```
Comment (Loquacity) [inline adoc/bp_client_migration.adoc]: Could these backticks be changed to explicit markup?

File: adoc/bp_client_migration.adoc [hunk 5]
```diff
@@ -47,192 +50,176 @@ endif::[]
...
 
 During the procedure, the machine reboots and performs the system upgrade.
-The process is controlled by {yast}
-and {ay}
-, not by plain [command]``zypper`` commands.
+The process is controlled by {yast} and {ay}, not by plain [command]``zypper`` commands.
```
Comment (Loquacity) [inline adoc/bp_client_migration.adoc]: remove 'plain'

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> Couple of notes inline. I think this could probably do with an editing
> pass to tidy up some of the language, but I'll hold off until you're
> happy with the content.

I'll stop for now and consider it done.

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg)

**Comment by Loquacity** [comment]
A note on these updates:

* It doesn't seem to be used much in our docs, but I can't recommend using non-breaking spaces enough. It stops things breaking over lines in horrible ways. They are great for between version names and numbers, and between numbers and their units of measurement. This style comes from the Chicago Guide. I haven't checked the brand gudelines for SUSE (must get on that), but most brands also say you can't split product names over lines, too, so we might night to consider that for our entities.
* There was a loooong procedure in this edit that had lots of sub-procedures (for want of a better term). I've split it into a process and procedures. So the process is the larger, overarching tasks, and then each procedure is a step in the process. It helps people keep track of the content as they're working through the steps, but it's also just easier for us to maintain. I have a talk somewhere that goes through procedure writing, and I'd be happy to go through it with the team if you think it would be useful. It's a learned skill :)

Anyway, I think you're good to merge now  :100:

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> * It doesn't seem to be used much in our docs, but I can't recommend
> using non-breaking spaces enough. It stops things breaking over lines in
> horrible ways. They are great for between version names and numbers, and
> between numbers and their units of measurement. This style comes from
> the Chicago Guide.

Yes, non-breaking spaces are very much appreciated :-) Instead of &nbsp;
or {nbsp} I prefer the plain unicode character: " " (not sure whether
you can see it).  It's not that intrusive.  In Emacs, I can enter rahter
painlessly.  And I set up the display engine to show me a reddish
underline (let's hope the attachment will make it through):



> I haven't checked the brand gudelines for SUSE (must get on that), but
> most brands also say you can't split product names over lines, too, so
> we might night to consider that for our entities.

Yeah, brands...  I do not like that rule, or: that a difficult rule.
Surely, we do not want "SUSE" to be hyphenated, but "SUSE Linux<nl>
Enterprise Server" is perfectly fine, IMO :-)

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg)

**Pull request merged**